### PR TITLE
Update ActiveMQ documentation

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update ActiveMQ documentation, Changes related to Helm Chart v3

Issue number: https://jira.kumoroku.com/jira/browse/SUMO-206934

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by swiatekm-sumo — COMMENTED** [review]
Can we also save the draw.io sources for these diagrams in this repo somewhere? WDYT @kimsauce ?

File: docs/integrations/containers-orchestration/activemq.md [hunk 1]
```diff
@@ -101,11 +101,13 @@ Choose your environment:
 
 <TabItem value="k8s">
 
-In Kubernetes environments, we use the Telegraf Operator, which is packaged with our Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from ActiveMQ in a Kubernetes environment. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Prometheus, Fluentd and FluentBit.
+In Kubernetes environments, we use the Telegraf Operator, which is packaged with our Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from ActiveMQ in a Kubernetes environment. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus and [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/activemq.md:104]:
```suggestion
In Kubernetes environments, we use the Telegraf Operator, which is packaged with our Kubernetes collection ([learn more](/docs/send-data/collect-from-other-data-sources/collect-metrics-telegraf/telegraf-collection-architecture)). The diagram below illustrates how data is collected from ActiveMQ in a Kubernetes environment. In the architecture shown below, there are four services that make up the metric collection pipeline: Telegraf, Telegraf Operator, Prometheus, and [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector).
```

File: docs/integrations/containers-orchestration/activemq.md [hunk 2]
```diff
@@ -101,11 +101,13 @@ Choose your environment:
...
+The first service in the metrics pipeline is Telegraf. Telegraf collects metrics from ActiveMQ. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment for example, Telegraf runs in the same pod as the containers it monitors. Telegraf uses the Jolokia2 input plugin to obtain metrics. (For simplicity, the diagram doesn’t show the input plugins.) The injection of the Telegraf sidecar container is done by the Telegraf Operator. Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
+
+In the logs pipeline, Sumo Logic Distribution for OpenTelemetry Collector collects logs written to standard out and forwards them to another instanced of Sumo Logic Distribution for OpenTelemetry Collector which enriches metadata and sends logs to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/activemq.md:110]:
```suggestion
In the logs pipeline, Sumo Logic Distribution for OpenTelemetry Collector collects logs written to standard out and forwards them to another instance of Sumo Logic Distribution for OpenTelemetry Collector, which enriches metadata and sends logs to Sumo Logic.
```

File: docs/integrations/containers-orchestration/activemq.md [hunk 3]
```diff
@@ -101,11 +101,13 @@ Choose your environment:
...
+The first service in the metrics pipeline is Telegraf. Telegraf collects metrics from ActiveMQ. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment for example, Telegraf runs in the same pod as the containers it monitors. Telegraf uses the Jolokia2 input plugin to obtain metrics. (For simplicity, the diagram doesn’t show the input plugins.) The injection of the Telegraf sidecar container is done by the Telegraf Operator. Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/activemq.md:108]:
```suggestion
The first service in the metrics pipeline is Telegraf. Telegraf collects metrics from ActiveMQ. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment for example, Telegraf runs in the same pod as the containers it monitors. Telegraf uses the Jolokia2 input plugin to obtain metrics. (For simplicity, the diagram doesn’t show the input plugins.) The injection of the Telegraf sidecar container is done by the Telegraf Operator. Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which enriches metadata and sends metrics to Sumo Logic.
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> Can we also save the draw.io sources for these diagrams in this repo somewhere? WDYT @kimsauce ?

It looks like @kkujawa-sumo did save this image to the repo: https://github.com/SumoLogic/sumologic-documentation/pull/1731/files

**Comment by kkujawa-sumo** [comment]
> Can we also save the draw.io sources for these diagrams in this repo somewhere? WDYT @kimsauce ?

Yes, it is good idea,  I had a doubt where to add sources from draw.io. 
I'll start adding them in same place where original pictures, with the same name but different extension.

**Pull request merged**